lib/Tooltip/index.js

```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import Popover from 'react-bootstrap/lib/Popover';
+
+const Tooltip = props => (
+  <Popover className={props.className} id={props.id} {...props}>
```

Comment:
Good catch! Realised later I had to actually spread all props but didn't remove the id 👍